src/TodoListItem.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styled from 'styled-components';
+import { VscTriangleDown, VscTriangleUp, VscClose } from 'react-icons/vsc';
+
+const TodoItem = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 200px;
+  height: 30px;
+`;
+
+const Buttons = styled.div`
+  display: flex;
+  flex-direction: row;
+`;
+
+const Button = styled.div`
+  margin: 2px;
+  &:hover {
+    cursor: pointer;
+    transform: scale(1.7);
+  }
+`;
+
+const TodoListItem = ({ todo, onToggle, onRemove }) => {
+  const { id, text, isDone } = todo;
+  return (
+    <TodoItem>
+      {text}
+      <Buttons>
+        <Button>
+          <div onClick={() => onToggle(id)}>
+            {isDone ? <VscTriangleUp /> : <VscTriangleDown />}
+          </div>
+        </Button>
```

Comment:
```javascript
const Button = styled.button`
  // styles for button component
`
```
```suggestion
        <Button onClick={() => onToggle(id)}>
            {isDone ? <VscTriangleUp /> : <VscTriangleDown />}
        </Button>
```
이런식으로 `Button` 컴포넌트를 `button`으로 선언해주신 다면 바로 컴포넌트에 대해 `onClick`을 바인딩할 수 있겠죠?

Reply:
Styled-component로 만든 컴포넌트에 prop을 바로 전달할 수 있다는 사실을 눈으로만 읽고 적용할 생각을 안했었네요.. 좋은 지적 감사합니다!!